demo/widgets/build.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable import/no-extraneous-dependencies
+process.env.NODE_ENV = 'production';
+// eslint-disable-next-line
+const Bundler = require('parcel-bundler');
+const { join } = require('path');
+const fs = require('fs');
+
+const { NETLIFY_REPO_DIR, NETLIFY_BUILD_BASE } = process.env;
+const cacheDir = NETLIFY_BUILD_BASE
+  ? join(NETLIFY_BUILD_BASE, 'cache', 'widgets')
+  : join(NETLIFY_REPO_DIR || '.', 'node_modules', '.cache');
```

Comment:
@erquhart what you think about this?

Reply:
Since those aren't documented I'd stay away from using them.

Reply:
Got it. Thanks